batchflow/tests/conftest.py

```diff
@@ -1,5 +1,36 @@
 """ Pytest configuration. """
 # pylint: disable=invalid-name, unused-import
 import pytest
+import numpy as np
+
+from batchflow import Dataset, ImagesBatch
+
 
 from .config_pass_test import single_config, multi_config, model_and_config
+
+
+@pytest.fixture()
+def model_setup_images_clf():
+    """ Pytest fixture to generate fake dataset and model config for image classification
+
+    Returns
+    -------
+    tuple
+        an instance of Dataset
+        a model config
+    """
+    def _model_setup(dataset_size=50, image_shape=(2, 100, 100), num_classes=10):
```

Comment:
You can add `data_format` parameter to transpose `image_shape`. That would allow to use this fixture in `dataformat_test` as well.

Reply:
Actually, that would remove the need for `image_shape` parameter as well